SU-HSDO/suhumsci
# SHS-5504: Prevent JS error when link is absent in vertical linked cards

# READY FOR REVIEW

## Summary
- Prevent JS error when link is absent in vertical linked cards
- Add prettier config to match eslint

## Need Review By (Date)
03/08

## Urgency
medium

## Steps to Test
1. Create a flexible page.
2. Add two or more postcards with the "Style" field set to "Vertical Linked Card". Leave the "Link" field empty in one of them.
3. Save the page. Confirm that there are no errors in the javascript console.
4. Confirm that the postcards with links work as expected.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
